beancount_reds_importers/libreader/csvreader.py

```diff
@@ -72,7 +72,7 @@ def initialize_reader(self, file):
             #     print(self.header_identifier, file.head())
 
     def deep_identify(self, file):
-        return re.match(self.header_identifier, file.head())
+        return re.match(self.header_identifier, file.head(), flags=re.DOTALL)
```

Comment:
This will actually affect a bunch of other importers when done here. Instead, simply override `deep_identify()` in your fidelity importer with `DOTALL` :)

Reply:
I went ahead and did this, hope that's okay :).

Reply:
Of course, makes sense and thank you!